test/defineContainer.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { defineContainer } from '../src/utils/defineContainer';
```

Comment:
Looks like test files aren't being linted, I see 4 char indents and missing semicolons